src/execution_plan/ops/shared/update_functions.h

```diff
@@ -33,6 +33,8 @@ void EvalEntityUpdates
 	GraphContext *gc,
 	dict *node_updates,
 	dict *edge_updates,
+	RG_Matrix *add_labels,
+	RG_Matrix *remove_labels,
```

Comment:
I'm not convinced we need `RG_Matrix` maybe we can use a `HYPER-SPARSE GrB_Matrix`